dependencies.gradle

```diff
@@ -20,12 +20,9 @@ ext {
     timberVersion = '4.7.1'
 
     // Android libraries
-    materialVersion = '1.0.0'
-    androidXAppcompatVersion = '1.1.0'
-    constraintLayoutVersion = '2.0.0-beta2'
+    materialVersion = '1.1.0'
+    constraintLayoutVersion = '2.0.0-beta7'
 
     // Beacon
-    beaconCoreVersion = '2.1.0'
-    beaconUiVersion = '2.1.0'
-
+    beaconVersion = '2.2.0'
```

Comment:
Is the idea to merge this now if so should we keep the  beaconVersion as 2.1.0? - not super strong feelings on this just trying to avoid merging with red CI.

Reply:
I was planning on holding off merging this until we actually release the artifact that discontinues `beacon-core` which if I'm not mistaken, will be 2.2.0. By then, Travis should green light this.